src/LatteContext/Collector/VariableCollector.php

```diff
@@ -64,7 +68,7 @@ public function collectData(Node $node, Scope $scope): ?array
         $isVariablesNode = false;
         $collectedVariables = [];
         foreach ($this->variableCollectors as $variableCollector) {
-            if (!$variableCollector->isSupported($node)) {
+            if (!in_array(get_class($node), $variableCollector->getNodeTypes())) {
```

Comment:
This is not the same as instanceof

Reply:
True. it could be child type. So we should iterate over node types and use instanceof

Reply:
this is fixed? if so, we can merge